man/resolved.conf.xml

```diff
@@ -264,6 +264,25 @@
         in use.</para></listitem>
       </varlistentry>
 
+      <varlistentry>
+        <term><varname>DNSStubListenerExtra=</varname></term>
+        <listitem><para>Takes a boolean argument or one of <literal>udp</literal> and <literal>tcp</literal>. If
+        <literal>udp</literal>, a DNS stub resolver will listen for UDP requests on address configured in
+        <varname>DnsStubListenerAddress=</varname>. If <literal>tcp</literal>, the stub will listen for TCP requests
+        on the same address and port. If <literal>yes</literal> (the default), the stub listens for both UDP and TCP requests.
+        If <literal>no</literal>, the stub listener is disabled</para></listitem>
+      </varlistentry>
+
+      <varlistentry>
+        <term><varname>DnsStubListenerAddress=</varname></term>
```

Comment:
`DnsStubListenerAddress=` --> `DNSStubListenerAddress=` to match spelling of the other settings.